wiki/items/armor.md

```diff
@@ -1,5 +1,178 @@
 ---
 title: Adding an Armor Set
 index: 3
-draft: true
----
\ No newline at end of file
+---
+# Adding an Armor Set
+
+This tutorial will assume that you already understand [Creating your First Item](first-item)
+
+## Creating an Armor Material
+
+To add an armor set, you start by creating an armor material. It defines the basic properties of your armor, such as the durability, the armor value, or the toughness.
+For this you create a new armor material class:
+
+`src/main/com/example/example_mod/ExampleArmorMaterial`:
+
+```java
+public class ExampleArmorMaterial implements ArmorMaterial {
```

Comment:
I would also mention you can make this an enum (like minecraft) if you have multiple material types.